util/index.ts

```diff
@@ -22,3 +23,15 @@ export const getChain = (chainId: number) => {
   }
   throw new Error(`no chain found at chainId ${chainId}`);
 };
+
+export const getBlockExplorerUrl = (txHashOrAddress: string, chainId: number) => {
+  const group = isHex(txHashOrAddress) ? (txHashOrAddress.length === 42 ? 'address' : 'tx') : 'ens';
```

Comment:
we don't yet have a need for this method to link out to addresses; i'd encourage against premature optimization here and leave it as tx only with the simplest code possible.

that being said it'd be good to have the code around if the need arises. At that time we'd probably want to create a new function rather than multipurpose this one.